projects/github-actions/repo-gardening/src/tasks/triage-issues/ai-labeling.js

```diff
@@ -94,6 +94,27 @@ Example response format:
 	return { labels, explanations: parsedResponse };
 }
 
+/**
+ * Clean up the issue content for OpenAI processing.
+ * Remove links from the issue content.
+ *
+ * @param {string} content - Issue body content.
+ *
+ * @return {string} Cleaned up issue content.
+ */
+function cleanIssueContent( content ) {
+	// Remove links in the format [link text](url), but keep the link text.
+	content = content.replace( /\[(.*?)\]\(https?:\/\/\S+\)/g, '$1' );
```

Comment:
Edge case: if the content has something like `text [xyz] text [foo](https://example.com) text`, it'll unexpectedly get turned into `text xyz] text [foo text`.

Another one: `text [foo](https://example.com)-[bar](https://example.com) text` (i.e. no space between the links) gets unexpectedly turned into `text foo text` rather than `text foo-bar text`.

Reply:
Ah yes, I need to be more exact. What do you think of bcf074a00ed7dc4aa55803cf860a5b6217714b24?